test/test_autograd.py

```diff
@@ -44,10 +43,7 @@
                                                         deviceCountAtLeast, skipCUDAIfCudnnVersionLessThan,
                                                         skipCUDAIf, skipMeta)
 
-
-# load_tests from common_utils is used to automatically filter tests for
-# sharding on sandcastle. This line silences flake warnings
-load_tests = load_tests
+import autograd.test_complex
```

Comment:
nit:

I would move this import at the bottom next to the runner. Or at least make a comment here explaining that this import has some significant side effect

Reply:
i moved it and added comments.